VEDA/django_secrets.py

```diff
@@ -5,25 +5,16 @@
 
```

Comment:
@muhammad-ammar I think we've decided to remove this?

Reply:
This is removed in https://github.com/edx/edx-video-pipeline/pull/50